src/Symfony/Component/ExpressionLanguage/Tests/ExpressionLanguageTest.php

```diff
@@ -164,6 +164,15 @@ public function testRegisterAfterEval($registerCallback)
     }
 
     /**
+     * @expectedException \RuntimeException
```

Comment:
please add also an `@expectedExceptionMessage` annotation

Reply:
Done. I did not add before to be consistent with the codebase in this component. I use `@expectedExceptionMessageRegExp` instead :)